plugins/inputs/ethtool/ethtool_linux.go

```diff
@@ -282,15 +282,15 @@ func (c *CommandEthtool) Interfaces(includeNamespaces bool) ([]NamespacedInterfa
 				Log:    c.Log,
 			}
 			if err := c.namespaceGoroutines[namespace].Start(); err != nil {
-				c.Log.Errorf(`Failed to start goroutine for namespace "%s": %s`, namespace, err)
+				c.Log.Errorf("Failed to start goroutine for namespace %q: %w", namespace, err)
 				delete(c.namespaceGoroutines, namespace)
 				continue
 			}
 		}
 
 		interfaces, err := c.namespaceGoroutines[namespace].Interfaces()
 		if err != nil {
-			c.Log.Warnf(`Could not get interfaces from namespace "%s": %s`, namespace, err)
+			c.Log.Warnf("Could not get interfaces from namespace %q: %w", namespace, err)
```

Comment:
`%w` should be only used for creating error, not for printing. Use `%s` or `%v` instead.